src/Pages/UserManager/EditUserInfo.js

```diff
@@ -14,19 +14,19 @@ import { sendVerificationEmail } from '../../APIFunctions/Mailer';
 export default function EditUserInfo(props) {
   const [firstName, setFirstName] = useState('');
   const [lastName, setLastName] = useState('');
-  const [password, setPassword] = useState(null);
+  const [password, setPassword] = useState('');
   const [doorCode, setDoorCode] = useState('');
-  const [major, setMajor] = useState('');
+  const [major, setMajor] = useState();
   const [pagesPrinted, setPagesPrinted] = useState();
   const [emailVerified, setEmailVerified] = useState();
-  const [accessLevel, setAccessLevel] = useState();
-  const [email, setEmail] = useState();
+  const [accessLevel, setAccessLevel] = useState('');
+  const [email, setEmail] = useState('');
   const [emailOptIn, setEmailOptIn] = useState();
   const [
     numberOfSemestersToSignUpFor,
     setNumberOfSemestersToSignUpFor
   ] = useState();
-  const [discordId, setDiscordId] = useState();
+  const [discordID, setDiscordID] = useState('');
```

Comment:
do any of these lines even need to be changed?

Reply:
done, was just messing around with those values and forgot to change back.